documentation/book/assembly-kafka-bridge-concepts.adoc

```diff
@@ -5,12 +5,12 @@
 [id='kafka-bridge-concepts-{context}']
 = Kafka Bridge
 
-This chapter provides an overview of the {ProductName} Kafka Bridge and helps you get started using its REST API to interact with {ProductName}. To try out the Kafka Bridge in your local environment, see the xref:assembly-kafka-bridge-quickstart-{context}[] later in this chapter.
+This chapter provides an overview of the {ProductName} Kafka Bridge on {ProductPlatformName} and helps you get started using its REST API to interact with {ProductName}. To try out the Kafka Bridge in your local environment, see the xref:assembly-kafka-bridge-quickstart-{context}[] later in this chapter.
```

Comment:
Shouldn't we use `Kubernetes` now directly here instead of `{ProductPlatformName}`?

Reply:
You're right, we should. I'll update.